style.css

```diff
@@ -0,0 +1,262 @@
+  align-items: center;
+  background-color: #191765;
+}
+
+main {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #191765;
+  background-image: url(./assets/images/eclipes.png);
+  background-repeat: no-repeat;
+  background-position-x: -200px;
+}
+
+header a {
+  font-family: Roboto, sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #fff;
+  text-decoration: none;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.hamburger {
+  background-color: #191765;
+  border: #191765;
+  padding-right: 24px;
+}
+
+.hamburger :hover {
+  background-color: #e5e5e5;
+}
+
+.hamburger:focus {
+  outline: none;
+  box-shadow: 0 0 0 3px #7d73fc;
+}
+
+.hamburger:active {
+  background-color: #e5e5e5;
+  border-color: #acb7c3;
+  color: #acb7c3;
+}
+
+.profile-container h1 {
+  font-family: Roboto, sans-serif;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 72px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+  color: #7d73fc;
+}
+
+.profile-container h2 {
+  font-family: Roboto, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #edecff;
+}
+
+.profile-container p {
+  font-family: Roboto, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #c8c8d4;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 40px;
+}
+
+.skill-stack {
+  height: 462.1505432128906px;
+  width: 303px;
+  left: 12px;
+  top: 235px;
+  border-radius: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.skill-stack h3 {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  color: #9490ff;
+}
+
+li {
+  list-style: none;
+}
+
+.skill-stack li {
+  font-family: Roboto, sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.social-media {
+  height: 104px;
+  width: 200px;
+  left: 87.5px;
+  top: 1092.1505126953125px;
+  border-radius: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.social-media-icon {
+  height: 40px;
+  width: 200px;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: #fff;
+  padding-top: 48px;
+}
+
+.experience-title {
+  padding-top: 83.85px;
+}
+
+.experience-title h2 {
+  font-family: Roboto, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #9490ff;
+}
+
+.experience-container > img {
```

Comment:
- [ ] Please make sure that there is one `<section>` tag is wrapping the whole work section. _Kindly see the hint I left in the above comment._

- [ ] However, make sure that you're following the given design in all aspects (_See above comment for more details._) You should make the background image of the work section as the given design.
Please refer to the design template, then make the corresponding changes.

- [ ] Following the two previous required changes, please keep in mind that it's mandatory in the [project's requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) to use both CSS grid and flexbox to align the work's section.
**Hint**
```
.work-section { // the whole container
  display: grid;
}

.project { // single project's container
  display: flex;
}
```